docs/config.c

```diff
@@ -3623,9 +3624,9 @@
 { "pop_last", DT_BOOL, false },
 /*
 ** .pp
-** If this variable is \fIset\fP, NeoMutt will try to use the "\fCLAST\fP" POP command
+** If this variable is \fIset\fP, NeoMutt will try to use the "\fILAST\fP" POP command
```

Comment:
@g-branden-robinson what do you think of this one?  I would make it bold, but I'm not entirely sure.